.github/workflows/main.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
 
       - name: Run chart-testing (install)
-        run: ct install
+        run: ct install --chart-repos bitnami=https://charts.bitnami.com/bitnami --remote origin --target-branch main
```

Comment:
If the purpose of this PR is to guarantee that arguments to all `ct` calls match, then we should reuse instead of copying.

For example, by creating a `CT_ARGS` env variable in the job.